network/http/detectors-http.tf

```diff
@@ -20,15 +20,15 @@ EOF
 }
 
 resource "signalfx_detector" "http_code_matched" {
-  name = format("%s %s", local.detector_name_prefix, "HTTP code ")
+  name = format("%s %s", local.detector_name_prefix, "HTTP code")
 
   program_text = <<-EOF
-    signal = data('http.code_matched', filter=${module.filter-tags.filter_custom})${var.http_code_matched_aggregation_function}${var.http_code_matched_transformation_function}.publish('signal')
+    signal = data('http.code_matched', filter=${module.filter-tags.filter_custom}, rollup='max')${var.http_code_matched_aggregation_function}${var.http_code_matched_transformation_function}.publish('signal')
     detect(when(signal < 1, lasting='${var.http_code_matched_lasting_duration_seconds}s', at_least=${var.http_code_matched_at_least_percentage})).publish('CRIT')
 EOF
 
   rule {
-    description           = "does not match"
+    description           = "does not match expected"
```

Comment:
We may put "does not match expected HTTP code" ?

Reply:
The alert message is concatenated with the detector name. Currently, the full message will be "HTTP code does not match expected"

Reply:
so we can put "does not match the expecting one" if you prefer ^^